pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: floating_action_bubble
-description: A animated menu using a floating action button , the aesthetic of the menu items are fully customisable 
+description: A animated menu using a floating action button , the aesthetic of the menu items are fully customisable
 version: 1.1.2
 homepage: https://github.com/Darshan0/floating_action_bubble
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
```

Comment:
Since this library relied on the very old Flutter SDK, I have raised the lower limit to `2.12.0`, the minimum required to be inspectable correctly by `flutter_lints`.